content/blog/2024-06-18-why-you-should-build-on-ethereum-classic-etc-is-proof-of-work/index.md

```diff
@@ -0,0 +1,101 @@
+
+Additionally, it is not enough for your dapp to be trust minimized by design if the underlying platform on which it is running is not. 
+
+## Trust Minimization Enables Censorship Resistance
+
+For example, you could actually build your app to work the same way on AWS, Azure, or Google Cloud, but you know that those services are centralized and captured by government and special interests, meaning that your app may be censored whenever they wish.
+
+Trust minimization is the opposite of censorship. 
+
+Because the dependency on trusted third parties is significantly reduced, then there is no entity or team that controls the system and, therefore, there is no way that government or special interests can capture it.
+
+This is very powerful because it does not only entail that your users will not be prevented from transacting with your dapps, but you as a developer will not be restricted when deploying your smart contracts or launching them.
+
+## Trust Minimization Enables Permissionlessness
+
+On ETC, not only will user transactions not be censored when sent to the blockchain, but creating and holding addresses and assets on various dapps by the public will be permissionless.
+
+Permissionlessness means that users don’t have to fill out forms, give their IDs, or pass background checks just to have an account on the ETC blockchain and interacting with your dapps.
+
+And, again, this also applies to your team. Sending smart contracts to ETC is completely permissionless.
+
+Indeed, today developer teams from all over the world have deployed dapps on ETC without barriers imposed by trusted thirds parties.
+
+## Only Proof of Work Is Decentralized
+
+The important thing to understand of all this is that only the technology of proof of work can enable decentralization in a blockchain on a global scale.
+
+Proof of work based Nakamoto Consensus in ETC enables all nodes and miners around the world to be synchronized on the same state of the blockchain every 13 seconds without the need to consult or check with any trusted third party!
+
+This means that anyone can join and leave the network running a node or mining whenever they wish. 
+
+This is the key to decentralization, and decentralization means trust minimization, and trust minimization means censorship resistance and permissionlessness for your dapps and users.
+
+## Proof of Work Makes Your Dapps Safer
+
+Because of all of these reasons, a POW blockchain as ETC is the only environment where your dapps will be safe.
+
+If your dapps are safe in ETC, then your users can rest assured that their money and interactions with your dapps will be safe.
+
+The problem with proof of stake (POS) and proof of authority (POA) systems such as Ethereum, Solana, and Ripple is that today they may seem decentralized, but the truth is that by doing away with POW they have eliminated the decentralization that enables trust minimization.
+
```

Comment:
what about when Ethereum used to be POW? why did that fail?

Reply:
This is interesting because I think that it didn't fail:

1. If you assume that ETC is the original chain, then it has been immutable since it was launched in 2015, so it has never been tampered with. Ethereum is just a fork that decided to change the rules several times (TheDAO fork, POS, EIP-1559, etc.).

2. If you assume that Ethereum is the original chain but tampered with systematically and mutable, then we can declare pow-based-nakamoto-consensus a failure or reason that maybe it was small at the time and easily manipulable. But, large POW chains are really immutable, e.g. Bitcoin.